### PR TITLE
Redux@1.0.0-rc

### PR DESCRIPTION
**Pull request by keyz**
See #8.

### CODE REVIEW

**Comment by emmenko** [comment]
@keyanzhang btw, I'm not 100% sure using `immutable-js` in this example is really necessary...

**Comment by gaearon** [comment]
I agree we don't need Immutable in this example.

**Comment by keyz** [comment]
Hi @emmenko and @gaearon,

Thanks for reviewing this! It makes sense to not adding `Immutable.js` to the example. Please check the latest commit.

**Comment by keyz** [comment]
Upgraded to `1.0.0-rc`. Thanks for your work, @gaearon!

File: package.json [hunk 1]
```diff
@@ -24,11 +24,13 @@
   "dependencies": {
     "babel-runtime": "5.6.15",
     "classnames": "2.1.2",
+    "immutable": "^3.7.4",
```
Comment (gaearon) [inline package.json]: Leftover?

Comment (keyz) [inline package.json]: fixed. also removed `react-pure-render`.

**Pull request closed without merging**